nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/event/pipe/queue/QueueBaseEventBundlePipe.java

```diff
@@ -64,9 +61,8 @@ public void initPipe(String name, Map<String, String> params) {
                 log.error("Unable to read batchSize parameter", e);
             }
         }
-
         queue = new ConcurrentLinkedQueue<>();
-        consumerTPE = new ThreadPoolExecutor(1, 1, 60, TimeUnit.MINUTES, new LinkedBlockingQueue<Runnable>());
+        consumerTPE = new ThreadPoolExecutor(1, 1, 60, TimeUnit.MINUTES, new LinkedBlockingQueue<>());
```

Comment:
The thread pool executor is already owning a queue for transfering events to the threads. If we want to keep making use of a thread pool we need twice. Once for collecting bundles and stacking them and another one for consuming with a bulk strategy. 

If not, we could keep the current design by just allocating a thread dedicated for the queue consuming.

In any case, we should double check the shutdown. Is the bundle consuming payload during the shutdown is acceptable  ? If not, how can we insure the bundles being recovered ?